src/components/templates/PosterHall/PosterHall.tsx

```diff
@@ -39,8 +47,52 @@ export const PosterHall: React.FC<PosterHallProps> = ({ venue }) => {
     ));
   }, [posterVenues]);
 
+  const [liveVenueIds, setLiveVenueIds] = useState<string[]>();
+
+  const { relatedVenues } = useRelatedVenues({
+    currentVenueId: venue.id,
+  });
+
+  const subVenues = useMemo(() => {
+    return relatedVenues.filter(
+      (relatedVenue) =>
+        relatedVenue.parentId === venue.id &&
+        relatedVenue.template !== VenueTemplate.posterpage
+    );
+  }, [relatedVenues, venue]);
+
+  const subVenueIds = useMemo(() => subVenues.map((venue) => venue.id), [
+    subVenues,
+  ]);
+
+  const { events: subVenueEvents } = useVenueEvents({
+    venueIds: subVenueIds,
+  });
+
+  useInterval(() => {
+    setLiveVenueIds(
+      subVenueEvents
+        .filter((event) => isEventLive(event))
+        .map((event) => event.venueId)
+    );
+  }, POSTERHALL_SUBVENUE_STATUS_MS);
+
+  const renderedSubvenues = useMemo(() => {
+    if (liveVenueIds)
+      return subVenues
+        .filter((subVenue) => liveVenueIds.includes(subVenue.id))
+        .map((subVenue) => (
+          <PosterPreview
+            key={subVenue.id}
+            posterVenue={subVenue as WithId<PosterPageVenue>}
```

Comment:
Since subVenue is not same as PosterPage, it's very error-prone approach

Reply:
I haven't looked too deeply into this, but I assume the issue is because `subVenues` (which comes from `relatedVenues`) isn't guaranteed to be a `PosterPage`.

Yet we could guarantee that by adding a check into the `.filter` above that makes the `subVenues`, to ensure it only includes `template === VenueTemplate.posterpage`

Then you should be able to just change the type on `subVenues` without using `as` to the appropriate type, and have it 'just work' down here.

Reply:
The only issue with the filtering approach is that the actual venue types for the current subVenues are 'audience', to avoid the space limits of the posterpage template. I have 5 of them set up for this, and have added in the `poster` fields to ensure they'll display properly as posterpreview cards in the posterhall, but if `subVenues` are filtered for template `posterpage` these will be filtered out.

Reply:
I've addressed this issue by adding a component `SubVenuePreview` 783cd299733235039740168864e5ebb3ed5a6bc0
which renders any venues that are not `=== VenueTemplate.posterpage` and that contain live events

Reply:
@mike-lvov FYI. The use of audience templates/etc here sounds.. odd to say the least..

Reply:
We have 4-5 parallel, live 'Poster Highlights' sessions that run during the first half hour of each of the poster sessions. These are 30-min moderator-guided presentations of 10-12 posters with summary videos. The aim of these sessions to to show folks some of the exciting posters to go and visit. These are not posters, but are rather talks related to the poster session. I anticipate we may have upwards of 100 attendees at some of these sessions. 

The initial idea was to put them in the symposium halls (audience venues), but we were told by the AV company that a 15 min break would be necessary between sessions for us to use those. 

The solution I came up with was to have venue cards appear at the top of the poster hall in a unique card style when these Poster Highlights events are live.

In brief I know this is a very OHBM-specific feature, but it fits quite well with what we're trying to achieve with these sessions. If there's a better way to implement, I'm open to that.